src/error.ts

```diff
@@ -0,0 +1,23 @@
+import {
+  OctokitResponse,
+  request as Request
+} from "@octokit/request/dist-types/types";
+import { GraphQlQueryResponse } from "./types";
+
+export class GraphqlError<T extends GraphQlQueryResponse> extends Error {
+  public request: Request;
```

Comment:
`error.request` is not the `@octokit/request` method, but request options (what `@octokit/endpoint` returns, an object with just `method`, `url`, `headers` and optionally `body` and `request`